tests/csapi/sync_test.go

```diff
@@ -345,6 +345,11 @@ func TestSync(t *testing.T) {
 			numResponsesReturned := 0
 			start := time.Now()
 			t.Logf("Will sync with since=%s", nextBatch)
+
+			// This part of the test is flaky for workerised Synapse with the default 5 second timeout,
+			// so bump it up to 10 seconds.
+			alice.SyncUntilTimeout = 10 * time.Second
```

Comment:
`alice` gets set up from outside the test. Will this change leak into other tests running in parallel?

Reply:
Yes, it will. In practice, this probably isn't much of a problem though.